install.sh

```diff
@@ -0,0 +1,355 @@
+
+set -e
+
+if [[ -t 1 ]]; then
+  tty_escape() { printf "\033[%sm" "$1"; }
+else
+  tty_escape() { :; }
+fi
+tty_mkbold() { tty_escape "1;$1"; }
+tty_blue="$(tty_mkbold 34)"
+tty_red="$(tty_mkbold 31)"
+tty_bold="$(tty_mkbold 39)"
+tty_reset="$(tty_escape 0)"
+
+ohai() {
+  printf "${tty_blue}==>${tty_bold} %s${tty_reset}\n" "$1"
+}
+
+warn() {
+  printf "${tty_red}Warning${tty_reset}: %s\n" "$1" >&2
+}
+
+abort() {
+  printf "${tty_red}Error${tty_reset}: %s\n" "$1" >&2
+  exit 1
+}
+
+OS="$(uname -s)"
+ARCH="$(uname -m)"
+
+if [[ "$ARCH" == "x86_64" ]]; then
+  ARCH="x86_64"
+elif [[ "$ARCH" == "amd64" ]]; then
+  ARCH="x86_64"
+elif [[ "$ARCH" == "arm64" ]]; then
+  ARCH="arm64"
+elif [[ "$ARCH" == "aarch64" ]]; then
+  ARCH="arm64"
+else
+  abort "Unsupported architecture: $ARCH"
+fi
+
+if [[ "$OS" == "Darwin" ]]; then
+  OS="Darwin"
+  EXTENSION="tar.gz"
+  INSTALL_DIR="/usr/local/bin"
+  
+  if command -v brew >/dev/null 2>&1 && [[ "$NO_BREW" != "true" ]]; then
+      ohai "Homebrew detected! Installing Agentuity CLI using Homebrew..."
+      if [[ "$VERSION" != "latest" ]]; then
+        warn "Version specification is not supported with Homebrew installation."
+        warn "Installing latest version instead. If you need a specific version,"
+        warn "use the --no-brew flag to use direct installation."
```

Comment:
we push every version for brew so you can install older or specific versions

see https://github.com/agentuity/homebrew-tap